configuration/micro_kernel_trait.rst

```diff
@@ -257,9 +297,7 @@ has one file in it::
 
     class MicroController extends AbstractController
     {
-        /**
-         * @Route("/random/{limit}")
-         */
+        #[Route('/random/{limit}')]
```

Comment:
Please revert this change. We should make the change on 5.4 and we decided to always show both annotation and attribute for now (using a `.. configuration-block::`).

Reply:
Reverted.